src/Parser/DeathKnight/Blood/Modules/Features/BloodPlagueUptime.js

```diff
@@ -11,24 +11,38 @@ class BloodPlagueUptime extends Analyzer {
     enemies: Enemies,
   };
 
+  get Uptime() {
+    return this.enemies.getBuffUptime(SPELLS.BLOOD_PLAGUE.id) / this.owner.fightDuration;
+  }
+
+  get UptimeSuggestionThresholds() {
+    return {
+      actual: this.Uptime,
+      isLessThan: {
+        minor: 0.94,
+        average: 0.84,
+        major: .74,
+      },
+      style: 'percentage',
+    };
+  }
+
   suggestions(when) {
-    const bloodplagueUptime = this.enemies.getBuffUptime(SPELLS.BLOOD_PLAGUE.id) / this.owner.fightDuration;
-    when(bloodplagueUptime).isLessThan(0.95)
+    when(this.UptimeSuggestionThresholds.actual).isLessThan(this.UptimeSuggestionThresholds.minor)
```

Comment:
Specifying it like this is no longer necessary, you can pass the entire thresholds object to `when()` which would take care of setting all the other props.